lib/rucio/client/scopeclient.py

```diff
@@ -34,43 +42,77 @@ def add_scope(
             scope: str
     ) -> bool:
         """
-        Sends the request to add a new scope.
+        Create a new scope for an account.
+
+        Scopes represent the namespaces that own datasets and files. Once
+        created, the scope can be used in DIDs (e.g. ``<scope>:<dataset>``) to
+        make ownership and visibility explicit.
 
         Parameters
         ----------
-        account :
-            The name of the account to add the scope to.
-        scope :
-            The name of the new scope.
+        account
+            Name of the account that will own the scope.
+        scope
+            Name of the scope to create.
 
         Returns
         -------
-            True if scope was created successfully.
+        bool
+            Literal ``True`` when the scope is created successfully.
 
         Raises
         ------
         Duplicate
-            If scope already exists.
+            If the scope already exists for the account.
         AccountNotFound
-            If account doesn't exist.
+            If the provided account does not exist.
+
+        Examples
+        --------
+        ??? Example
+
+            Create a scope for the ``dataops`` account. If the scope already
+            exists for the account, a ``Duplicate`` exception is raised.
+
+            ```python
+            from rucio.client.scopeclient import ScopeClient
+
+            # The client will pick up authentication from the standard Rucio
+            # configuration (e.g. ~/.ruciorc or RUCIO_* environment variables).
+            client = ScopeClient()
+
+            client.add_scope(account="dataops", scope="user.test")
+            ```
         """
 
-        path = '/'.join([self.SCOPE_BASEURL, account, 'scopes', quote_plus(scope)])
+        path = '/'.join(
+            [self.SCOPE_BASEURL, account, 'scopes', quote_plus(scope)]
+        )
         url = build_url(choice(self.list_hosts), path=path)
         r = self._send_request(url, method=HTTPMethod.POST)
         if r.status_code == codes.created:
             return True
         else:
-            exc_cls, exc_msg = self._get_exception(headers=r.headers, status_code=r.status_code, data=r.content)
+            exc_cls, exc_msg = self._get_exception(
+                headers=r.headers,
+                status_code=r.status_code,
+                data=r.content
+            )
             raise exc_cls(exc_msg)
 
     def list_scopes(self) -> list[str]:
         """
-        Sends the request to list all scopes.
+        List all available scopes.
```

Comment:
"available" is sort of a keyword, I'm a little nervous this may imply there is a state attached to scopes where "unavailable" is also an option. Maybe - 
```suggestion
        List all scopes belonging to the client's authenticated account. 
```

Reply:
The trailing space is coming from here. Should I just send a new squashed and rebased fixed version?